### ISSUE TITLE
Mediapipe Web Pose Landmarker doesn't return visibility 

### ISSUE THREAD

**Issue by jays0606** [google/mediapipe#4409]
### Have I written custom code (as opposed to using a stock example script provided in MediaPipe)

None

### OS Platform and Distribution

Mac OS

### Task name (e.g. Image classification, Gesture recognition etc.)

Pose Estimation

### Programming Language and version (e.g. C++, Python, Java)

JavaScript

### Describe the actual behavior

```shell
It says in the doc mediapipe pose landmark should return visibility, but in reality the output only returns x, y, z.
```

### Describe the expected behaviour

```shell
The visibility 0.0 ~ 1.0 should be returned along with xyz.
```

### Standalone code/steps you may have used to try to get what you need

```shell
const poseLandmarker = await PoseLandmarker.createFromOptions(vision, {
        baseOptions: {
            modelAssetPath: `pose_landmarker_full.task`,
            delegate: "GPU",
        },
        runningMode: "VIDEO",
        numPoses: 1,
    });

And run poseLandmark.detectForVideos()
I printed out the results.
```

**Comment by Bilalismail** [comment]
Facing the same issue. I'm currently using Z values to determine visibility by scoring against right and left landmarks. Can you confirm if the visibility score will be coming back?

**Comment by emoc11** [comment]
Same issue here, could be greate to have it back again soon ! 

Thanks for your work :)

**Comment by Palashxeuron** [comment]
any updates

**Comment by Filiflo** [comment]
There has been a more recent post on similar issues.
They seem to still be working on it.

here are some references to posts that seem to have the same problem:
[typescript - same issue(last addressed 2 weeks from writing)](https://github.com/google/mediapipe/issues/4605)
[duplicate of this issue - has been marked as such](https://github.com/google/mediapipe/issues/4536#issuecomment-1592460079)

**Comment by kuaashish** [comment]
HI @jays0606,

Could you please verify the current status using the nightly version at https://www.npmjs.com/package/@mediapipe/tasks-vision/v/0.10.12-rc.20240407? It must return the visibility due to the merged https://github.com/google/mediapipe/pull/5142. These changes are expected to be included in the upcoming major release.

Thank you!!

**Comment by SinchanKM** [comment]
It's available for mediapipe web v0.10.11 released on 8th March 2024
